jenkins_pipelines/environments/uyuni-master-qe-build-validation

```diff
@@ -0,0 +1,48 @@
+            booleanParam(name: 'must_run_core', defaultValue: true, description: 'Run Core features'),
+            booleanParam(name: 'must_sync', defaultValue: true, description: 'Sync. products and channels'),
+            booleanParam(name: 'must_add_MU_repositories', defaultValue: true, description: 'Add MU channels'),
+            booleanParam(name: 'must_add_non_MU_repositories', defaultValue: true, description: 'Add non MU channels'),
+            booleanParam(name: 'must_add_keys', defaultValue: true, description: 'Add Activation Keys'),
+            booleanParam(name: 'must_create_bootstrap_repos', defaultValue: true, description: 'Create bootstrap repositories'),
+            booleanParam(name: 'must_boot_proxy', defaultValue: true, description: 'Bootstrap Proxy'),
+            booleanParam(name: 'must_boot_monitoring', defaultValue: true, description: 'Bootstrap Monitoring Server'),
+            booleanParam(name: 'must_boot_clients', defaultValue: true, description: 'Bootstrap clients'),
+            booleanParam(name: 'must_run_tests', defaultValue: true, description: 'Run Smoke Tests'),
+            booleanParam(name: 'must_prepare_retail', defaultValue: true, description: 'Prepare and run Retail'),
+            booleanParam(name: 'must_run_containerization_tests', defaultValue: true, description: 'Run Containerization Tests'),
+            booleanParam(name: 'confirm_before_continue', defaultValue: false, description: 'Confirmation button between stages'),
+            string(name: 'capybara_timeout', defaultValue: '30', description: 'Capybara max. waiting time'),
+            string(name: 'default_timeout', defaultValue: '300', description: 'Default timeout used in our Test Framework'),
+            text(name: 'custom_repositories', defaultValue: '{"server":{"server_stack":"","salt15sp3_base":"","salt15sp3_python2_module":"","salt15sp3_server_apps_module":"","extra_sle_that_affects_suma_srv":""},"proxy":{"salt":"","tools":"","salt15sp3":"","extra_sle_that_affects_suma_proxy":""},"sle12sp4_minion":{"salt":"","tools":""},"sle12sp4_client":{"tools":""},"sle12sp5_minion":{"salt":"","tools":""},"sle12sp5_client":{"tools":""},"sle15_minion":{"salt":"","tools":""},"sle15_client":{"tools":""},"sle15sp1_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp1_client":{"tools":""},"sle15sp2_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp2_client":{"tools":""},"sle15sp3_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp3_client":{"tools":""},"debian10_minion":{"salt":"","tools":""},"ubuntu1804_minion":{"salt":"","tools":""},"ubuntu2004_minion":{"salt":"","tools":""},"ceos7_minion":{"salt":"","tools":""},"ceos7_client":{"tools":""},"rocky8_minion":{"salt":"","tools":""},"rocky8_client":{"tools":""},"opensuse154arm_minion":{"salt":"","tools":""}}', description: 'Salt & Client Tools MU Repositories for each client, in json format')
```

Comment:
I'm curious, is uyuni not based on 15sp4? just wondering what the 15sp3 repos are doing here :thinking: feel free to disregard if it makes sense

Reply:
I am not sure about this either, I simply copied that over from the 4.3 BV file. I also see (and in 4.3 BV, too) that there is everything SLE Micro related missing.

Reply:
We discussed in the QE channel to use `{}` as default values since most of the values were outdated anyway.